tests/core_tests.py

```diff
@@ -291,7 +291,8 @@ def test_slices(self):
             ]
         for name, method, url in urls:
             logging.info(f'[{name}]/[{method}]: {url}')
-            self.client.get(url)
+            resp = self.client.get(url)
+            self.assertEqual(resp.status_code, 200)
```

Comment:
This is important! This is the test that runs every single example chart, and calls both `explore` and `explore_json` for each one. We were getting 500s before this, and tests were passing...